src/php/tests/interop/interop_client.php

```diff
@@ -36,6 +36,9 @@
 require 'message_set.php';
 require 'messages.php';
 require 'test.php';
+use Google\Auth\CredentialsLoader;
+use Google\Auth\ApplicationDefaultCredentials;
+use GuzzleHttp\ClientInterface;
```

Comment:
As for the class itself, I need it to match the signature of the auth library `update_metadata` function.